[36m2023-04-12 20:51:27[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mc.d.TakeOutApplication[0;39m - (Starting TakeOutApplication using Java 19 on LAPTOP-0RD4O4D6 with PID 8428 (D:\User\Desktop\Â§ñÂçñAPP\takeout-backend\target\classes started by User in D:\User\Desktop\Â§ñÂçñAPP\takeout-backend)
[36m2023-04-12 20:51:27[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mc.d.TakeOutApplication[0;39m - (No active profile set, falling back to 1 default profile: "default"
[36m2023-04-12 20:51:29[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mo.a.c.h.Http11NioProtocol[0;39m - (Initializing ProtocolHandler ["http-nio-8081"]
[36m2023-04-12 20:51:29[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mo.a.c.c.StandardService[0;39m - (Starting service [Tomcat]
[36m2023-04-12 20:51:29[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mo.a.c.c.StandardEngine[0;39m - (Starting Servlet engine: [Apache Tomcat/9.0.71]
[36m2023-04-12 20:51:29[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mo.a.c.c.C.[.[.[/][0;39m - (Initializing Spring embedded WebApplicationContext
[36m2023-04-12 20:51:29[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mc.z.h.HikariDataSource[0;39m - (HikariPool-1 - Starting...
[36m2023-04-12 20:51:30[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mc.z.h.HikariDataSource[0;39m - (HikariPool-1 - Start completed.
[36m2023-04-12 20:51:31[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mo.a.c.h.Http11NioProtocol[0;39m - (Starting ProtocolHandler ["http-nio-8081"]
[36m2023-04-12 20:51:31[0;39m [32m[main][0;39m [34mINFO [0;39m [1;35mc.d.TakeOutApplication[0;39m - (Started TakeOutApplication in 4.829 seconds (JVM running for 5.5)
[36m2023-04-12 20:52:07[0;39m [32m[http-nio-8081-exec-2][0;39m [34mINFO [0;39m [1;35mo.a.c.c.C.[.[.[/][0;39m - (Initializing Spring DispatcherServlet 'dispatcherServlet'
[36m2023-04-12 20:58:27[0;39m [32m[lettuce-nioEventLoop-4-1][0;39m [34mINFO [0;39m [1;35mi.l.c.p.CommandHandler[0;39m - (null Unexpected exception during request: java.net.SocketException: Connection reset
java.net.SocketException: Connection reset
	at java.base/sun.nio.ch.SocketChannelImpl.throwConnectionReset(SocketChannelImpl.java:401)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:434)
	at io.netty.buffer.PooledByteBuf.setBytes(PooledByteBuf.java:259)
	at io.netty.buffer.AbstractByteBuf.writeBytes(AbstractByteBuf.java:1132)
	at io.netty.channel.socket.nio.NioSocketChannel.doReadBytes(NioSocketChannel.java:357)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:151)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1589)
[36m2023-04-12 20:58:27[0;39m [32m[lettuce-eventExecutorLoop-1-4][0;39m [34mINFO [0;39m [1;35mi.l.c.p.ConnectionWatchdog[0;39m - (Reconnecting, last destination was localhost/127.0.0.1:6379
[36m2023-04-12 20:58:28[0;39m [32m[SpringApplicationShutdownHook][0;39m [34mINFO [0;39m [1;35mc.z.h.HikariDataSource[0;39m - (HikariPool-1 - Shutdown initiated...
[36m2023-04-12 20:58:28[0;39m [32m[SpringApplicationShutdownHook][0;39m [34mINFO [0;39m [1;35mc.z.h.HikariDataSource[0;39m - (HikariPool-1 - Shutdown completed.
